controllers/array_action/array_mediator_svc.py

```diff
@@ -1555,4 +1555,4 @@ def create_host(self, host_name, initiators, connectivity_type):
         self._mkhost(host_name, connectivity_type, ports)
 
     def delete_host(self, host_name):
-        raise NotImplementedError
+        self.client.svctask.rmhost(object_id=host_name)
```

Comment:
what if an error is raised, but its just a warning?

Reply:
done #561